packages/broker/src/ConfigWizard.ts

```diff
@@ -131,9 +130,6 @@ const PRIVATE_KEY_PROMPTS: Array<inquirer.Question | inquirer.ListQuestion | inq
     }
 ]
 
-export const getPrivateKey = (answers: inquirer.Answers): string => {
```

Comment:
When we reorganize the code, it is maybe easiest to choose one primary factor which guides the order. It could be anything, e.g.
- by the user flow (all prompting first, then config generation, then storing the config)
- by content (all private key handing first, then all plugin handling)
- ...

In all cases, it is good to have the very generic code at the top and the main at the end, as you currently have.

Reply:
the idea behind the code sorting is:
- imports
- helper methods
- static values
- inquirer prompts
- post-question logic handling
- start fn

Reply:
Sounds good. Then maybe:
- `PROMPTS` should be in the static values section
- `getPrivateKey` in helper methods (or locally near `start`)
- `selectStoragePath` and `createStorageFile` near each other